pyanaconda/ui/gui/spokes/keyboard.py

```diff
@@ -344,12 +344,20 @@ def completed(self):
         if flags.flags.automatedInstall and not self._seen:
             return False
 
-        # The currently activated layout is a different from the
-        # selected ones. Ignore VNC, since VNC keymaps are weird
-        # and more on the client side.
-        if not self._confirmed and not flags.flags.usevnc \
-                and self._xkl_wrapper.get_current_layout() not in self._l12_module.XLayouts:
-            return False
+        # Not confirmed by a user should we request the check?
+        if not self._confirmed:
```

Comment:
It could be definitely simplified.

```
if self._confirmed:
  return True

if flags.flags.usevnc:
  return True

if not keyboard.can_configure_keyboard():
  return True

return self._xkl_wrapper.get_current_layout() not self._l12_module.XLayouts
```

Reply:
Ha, I knew it could be done better. Thanks for suggestion, will implement that to master.